### DISCUSSION TITLE
Issue with the server for my css files

### DISCUSSION THREAD

**Discussion by simon-zeru** [community/community#47481] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,
Quick context : I'm developping a website for a project, and using VS code with live server, my website has no issues, and the page's style is looking great. 

I updated my repository with node.js, and added all files just like in my folder. But when I go on the page to see my site ; https://tiborz08.github.io/MySportProgram/ , the style isn't appearing so I guess there the server fail to read my css and js files...

I will send you the state of the console with this message.

![Capture d’écran 2023-02-14 à 21 54 38](https://user-images.githubusercontent.com/111081518/218863221-aab62f8f-309c-44bf-bf68-5b517f0892f4.jpg)

I hope that we can find a solution to fix this problem

Thank you for your time,
with kind regards

Simon

**Comment by emilburganov** [comment]
GitHub pages are only suitable for static sites, so node js won't work on them(

**Comment by airtower-luna** [comment]
Your `index.html` uses incorrect absolute paths. To pick [just one example](https://github.com/Tiborz08/MySportProgram/blob/b327fabb02b27f2aa51b033c86764116c11f9d11/index.html#LL9C1-L9C51):

```html
    <link href="/dist/style.css" rel="stylesheet">
```

Because of the leading slash, the browser will look for https://tiborz08.github.io/dist/style.css, but your stylesheet is actually deployed at https://tiborz08.github.io/MySportProgram/dist/style.css (within your site directory). The best solution would be to use relative paths (no leading slash, relative to the directory containing the file), alternatively make sure the absolute path is correct.

See also: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#absolute_urls_vs._relative_urls
